src/sentry/ingest/transaction_clusterer/datasource/redis.py

```diff
@@ -201,37 +199,28 @@ def record_span_descriptions(
 
 
 def _get_span_description_to_store(span: Mapping[str, Any]) -> Optional[str]:
-    if not span.get("op", "").startswith("http"):
+    if not span.get("op") in ("resource.css", "resource.js"):
```

Comment:
Is `resource.js` an op? I don't see it documented in [dev docs](https://develop.sentry.dev/sdk/performance/span-operations/#browser). Related -- should we include more `resource.*` spans?